src/virtualenv/activation/via_template.py

```diff
@@ -36,12 +36,8 @@ def replacements(self, creator, dest_folder):
         current_platform = sysconfig.get_platform()
         platforms = ["mingw", "cygwin", "msys"]
         if any(platform in current_platform for platform in platforms):
-            pattern = re.compile("^([A-Za-z]):(.*)")
-            match = pattern.match(str(creator.dest))
-            if match:
-                virtual_env = "/" + match.group(1).lower() + match.group(2)
-            else:
-                virtual_env = str(creator.dest)
+            code, out, err = run_cmd(["cygpath", str(creator.dest)])
```

Comment:
Also use _ and __ for the variable names you don't need/use; though you should raise here in error if the code is not zero, not? with the out and error attached to the error